src/main/resources/templates/product_details.html

```diff
@@ -12,7 +12,7 @@
             class="display-3 mb-5"
             th:text="${product.name} ?: 'Product Name'"
           ></h2>
-          <div class="row" th:utext="${product.description}"></div>
+          <div class="row" id="ProductDescription" th:utext="${product.description}"></div>
```

Comment:
In html, `id`, `class` and other data should be kebab cased.
```suggestion
          <div class="row" id="product-description" th:utext="${product.description}"></div>
```

Reply:
@Isammoc is it ok to leave id as `ProductDescription` in this case, else I need to re-record the video?

thanks!

Reply:
It's against best practices. But users will have to manage real applications with downsides too ^^